### ISSUE TITLE
Cross correlation Loss

### ISSUE THREAD

**Issue by ilijanikolov** [Y-debug-sys/Diffusion-TS#37]
Hello,

For the cross correlation loss, I was wondering why there was an extra factor of 10 in this function:

def compute(self, x_fake):
        cross_correl_fake = cacf_torch(self.transform(x_fake), 1).mean(0)[0]
        loss = self.norm_foo(cross_correl_fake - self.cross_correl_real.to(x_fake.device))
        return loss / 10.

Is this 10 hard coded? Should it be the number of features?

Thank you!

**Comment by Y-debug-sys** [comment]
Hi, this 10 is just used for scaling down the main results of our paper, and it has no other special meaning. The metirc's detailed description can be found in the paper [[Conditional-Sig-Wasserstein-GANs](https://arxiv.org/abs/2006.05421)], and we reuse its [official  code](https://github.com/SigCGANs/Conditional-Sig-Wasserstein-GANs/blob/master/lib/test_metrics.py). Thanks!
